### ISSUE TITLE
Please provide more note lengths for bpm selection

### ISSUE THREAD

**Issue by spaetz** [thetwom/toc2#42]
Hi,
my wife loves to play guitar and I love FOSS, so thanks for the metronome! (can I donate a little somewhere?)
One thing which annoys my wife is things like:

![Screenshot_2021-09-10 Die punktierte Viertel Note](https://user-images.githubusercontent.com/63132/132809938-33e4af5b-ecca-42f9-9140-6e5dbb1c85e4.png) = 73bpm

Would it be possible to add some of these to the selection of 1/4, 1/8, 1/16 = 120bpm?

I know one could calculate the right bpm and set it, but doing that kind of maths is beyond some musicians :-).

If you think it clutters the screen too much, that is perfectly fine. Thanks in any case!

**Comment by thetwom** [comment]
Thanks for your feedback!

I agree, that if you have such a scenario it is super annoying not to have this option.

In theory, there is enough "space" to implement this feature. But I am wondering, if we would give up consistency over the UI. Meaning, that if these kind of "beat durations" existed, user could automatically expect also to have these notes available. Or if  even more durations like triple-notes would be needed. And adding all this would of course make the UI more and more complex. We would at least have to provide some good reasons, why this is only available for the beat duration ...

I will give it another thought for some time. Unfortunately, I don't have time to implement this short term ...  

(there is no option to donate for this project :-)

**Comment by spaetz** [comment]
Thanks for the quick reply. I see your point. On the one hand, having frequent options available would be nice. On the other hand, the UI feels already quite "full" and needs some getting used to (I wonder if one could simplify the GUI a little by e.g. ditching the volume slider button on the main screen) (volume can be set by clicking on the single notes).

**Comment by thetwom** [comment]
Exactly. Problem about dropping options is, that this app does not collect any usage data (it never will),  so I can't say how much different functionalities are used ... and dropping options which are used a lot is not so nice :-). So we must be careful about introducing new options ....
